### PR TITLE
CrowdStrike FDR Host Inventory app doc

### PR DESCRIPTION
**Pull request by JV0812**
## Purpose of this pull request

This pull request is to create new CrowdStrike FDR Host Inventory app doc.

## Select the type of change

- [ ] Minor Changes - Typos, formatting, slight revisions
- [ ] Update Content - Revisions, updating sections
- [x] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - .clabot, version updates, maintenance, dependencies, new packages for the site (Docusaurus, Gatsby, React, etc.)

## Ticket (if applicable)

[CrowdStrike FDR Host Inventory App - Create Dochub Page](https://sumologic.atlassian.net/browse/DOCS-403)

### CODE REVIEW

File: blog-service/2024-08-23-apps.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+---
+title: CrowdStrike FDR Host Inventory (Apps)
+image: https://help.sumologic.com/img/sumo-square.png
+keywords:
+  - crowdStrike-fdr-host-inventory
```
Comment (kimsauce) [inline blog-service/2024-08-23-apps.md:5]:
```suggestion
  - crowdstrike
  - crowdstrike-fdr-host-inventory
```

File: docs/integrations/saas-cloud/crowdStrike-fdr-host-inventory.md [hunk 2]
```diff
@@ -0,0 +1,148 @@
+---
+id: crowdStrike-fdr-host-inventory
```
Comment (kimsauce) [inline docs/integrations/saas-cloud/crowdStrike-fdr-host-inventory.md:2]:
```suggestion
id: crowdstrike-fdr-host-inventory
```

File: cid-redirects.json [hunk 3]
```diff
@@ -2035,6 +2035,7 @@
   "/cid/4017": "/docs/manage/security/saml/set-up-saml",
   "/cid/4018": "/docs/send-data/installed-collectors",
   "/cid/4019": "/docs/send-data/installed-collectors/sources/script-action",
+  "/cid/4412": "/docs/integrations/saas-cloud/crowdStrike-fdr-host-inventory",
```
Comment (kimsauce) [inline cid-redirects.json:2038]:
```suggestion
  "/cid/4412": "/docs/integrations/saas-cloud/crowdstrike-fdr-host-inventory",
```

File: blog-service/2024-08-23-apps.md [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+  - apps
+hide_table_of_contents: true
+authors:
+  - url: https://help.sumologic.com/release-notes-service/rss.xml
+    image_url: /img/release-notes/rss-orange.png
+---
+
+We're excited to announce the release of the new CrowdStrike FDR Host Inventory app for Sumo Logic. This app leverages the Sumo Logic Cloud-to-Cloud CrowdStrike FDR Host Inventory source to gain actionable insights and visualizations to help you prioritize and mitigate device security incidents efficiently.
+
+Check out our technical documentation [here](docs/integrations/saas-cloud/crowdStrike-fdr-host-inventory) to learn how to set up and use the CrowdStrike FDR Host Inventory app for Sumo Logic.
```
Comment (kimsauce) [inline blog-service/2024-08-23-apps.md:15]:
```suggestion
Check out our technical documentation [here](docs/integrations/saas-cloud/crowdstrike-fdr-host-inventory) to learn how to set up and use the CrowdStrike FDR Host Inventory app for Sumo Logic.
```

File: sidebars.ts [hunk 5]
```diff
@@ -2292,6 +2292,7 @@ integrations: [
           'integrations/saas-cloud/cisco-umbrella',
           'integrations/saas-cloud/citrix-cloud',
           'integrations/saas-cloud/cloudflare',
+          'integrations/saas-cloud/crowdStrike-fdr-host-inventory',
```
Comment (kimsauce) [inline sidebars.ts:2295]:
```suggestion
          'integrations/saas-cloud/crowdstrike-fdr-host-inventory',
```

File: docs/integrations/saas-cloud/index.md [hunk 6]
```diff
@@ -87,6 +87,12 @@ Learn about the Sumo Logic apps for SaaS and Cloud applications.
   <p>Analyze logs, events, and trends from your websites and apps on the Cloudflare network.</p>
   </div>
 </div>
+<div className="box smallbox card">
+  <div className="container">
+  <a href="/docs/integrations/saas-cloud/crowdStrike-fdr-host-inventory"><img src={useBaseUrl('img/integrations/security-threat-detection/crowdstrike.png')} alt="icon" width="90"/><h4>CrowdStrike FDR Host Inventory</h4></a>
```
Comment (kimsauce) [inline docs/integrations/saas-cloud/index.md:92]:
```suggestion
  <a href="/docs/integrations/saas-cloud/crowdstrike-fdr-host-inventory"><img src={useBaseUrl('img/integrations/security-threat-detection/crowdstrike.png')} alt="icon" width="90"/><h4>CrowdStrike FDR Host Inventory</h4></a>
```

**Review by kimsauce — APPROVED** [review]

**Comment by jpipkin1** [comment]
@JV0812, I added the app to the product list (see comment "[Add app to product list](https://github.com/SumoLogic/sumologic-documentation/pull/4451/commits/fefd051f3af2fb1ca2887d782def25d6f214f44a)").

**Pull request merged**